packages/graphql/src/mutations/_txHelper.js

```diff
@@ -141,6 +167,22 @@ export default function txHelper({
     const shouldUseRelayer = useRelayer({ mutation, value })
     const shouldUseProxy = await useProxy({ proxy, addr, to, from, mutation })
     debug(`shouldUseProxy: ${shouldUseProxy}`)
+    debug(`shouldUseRelayer: ${shouldUseRelayer}`)
+
+    // Get our gas price
+    if (!shouldUseRelayer && ['mainnet', 'rinkeby'].includes(contracts.net)) {
+      if (window.ethereum) {
+        // Don't use eth_gasPrice for MM
+        try {
+          gasPrice = await fetchGasPrice()
+        } catch (err) {
+          debug(`Error fetching gas price`, err)
+        }
+      } else {
+        // But use it when we're using our provider
+        gasPrice = await web3.eth.getGasPrice()
```

Comment:
Is there a point in us having an EthGasStationProvider ?
Could we get rid of it and just always set the gasPrice here ?

Reply:
We could do this as a separate cleanup PR. Just asking for my own understanding... :)

Reply:
`web3.eth.getGasPrice` will use the provider(if it's there).  And so will all the services that don't use GraphQL.